core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -49,7 +49,8 @@ registry.ci.openshift.org/ocp-kni/4.17:cnf-tests quay.io/openshift-kni/cnf-tests
 registry.ci.openshift.org/ocp-kni/4.17:dpdk quay.io/openshift-kni/dpdk:4.17 quay.io/openshift-kni/dpdk:4.17.0
 registry.ci.openshift.org/ocp-kni/4.18:cnf-tests quay.io/openshift-kni/cnf-tests:4.18 quay.io/openshift-kni/cnf-tests:4.18.0
 registry.ci.openshift.org/ocp-kni/4.18:dpdk quay.io/openshift-kni/dpdk:4.18 quay.io/openshift-kni/dpdk:4.18.0
-registry.ci.openshift.org/ocp-kni/4.19:cnf-tests quay.io/openshift-kni/cnf-tests:4.19 quay.io/openshift-kni/cnf-tests:4.19.0 quay.io/openshift-kni/cnf-tests:latest
+registry.ci.openshift.org/ocp-kni/4.19:cnf-tests quay.io/openshift-kni/cnf-tests:4.19 quay.io/openshift-kni/cnf-tests:4.19.0
+registry.ci.openshift.org/ocp-kni/4.20:cnf-tests quay.io/openshift-kni/cnf-tests:4.20 quay.io/openshift-kni/cnf-tests:4.20.0 quay.io/openshift-kni/cnf-tests:latest
 registry.ci.openshift.org/ocp-kni/4.19:dpdk quay.io/openshift-kni/dpdk:4.19 quay.io/openshift-kni/dpdk:4.19.0 quay.io/openshift-kni/dpdk:latest
```

Comment:
don't we need to do the same for the dpdk image? 
it's also part of cnf-feature-deploy

Reply:
going back in the history it seems it is indeed needed. let me change that. although now there is a new dpdk project repo but I don't know about plans for cancelling dpdk in cnf-features-deploy.

Reply:
updated. Thanks!